tools/Azure.Mcp.Tools.StorageSync/tests/Azure.Mcp.Tools.StorageSync.LiveTests/StorageSyncCommandTests.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Mcp.Tests;
+using Azure.Mcp.Tests.Client;
+using Azure.Mcp.Tests.Client.Attributes;
+using Azure.Mcp.Tests.Client.Helpers;
+using Azure.Mcp.Tests.Generated.Models;
+using Xunit;
+
+namespace Azure.Mcp.Tools.StorageSync.LiveTests;
+
+public class StorageSyncCommandTests(ITestOutputHelper output, TestProxyFixture fixture) : RecordedCommandTestsBase(output, fixture)
+{
+    public override List<UriRegexSanitizer> UriRegexSanitizers => new[]
+    {
+        new UriRegexSanitizer(new UriRegexSanitizerBody
+        {
+            Regex = "resource[gG]roups\\/([^?\\/]+)",
+            Value = "Sanitized",
+            GroupForReplace = "1"
+        })
+    }.ToList();
+
+    public override List<GeneralRegexSanitizer> GeneralRegexSanitizers => new[]
+    {
+        new GeneralRegexSanitizer(new GeneralRegexSanitizerBody()
+        {
+            Regex = Settings.ResourceGroupName,
+            Value = "Sanitized",
+        }),
+        new GeneralRegexSanitizer(new GeneralRegexSanitizerBody()
+        {
+            Regex = Settings.ResourceBaseName,
+            Value = "Sanitized",
+        }),
+        new GeneralRegexSanitizer(new GeneralRegexSanitizerBody()
+        {
+            Regex = Settings.SubscriptionId,
+            Value = "00000000-0000-0000-0000-000000000000",
+        })
+    }.ToList();
+
+    public override List<BodyRegexSanitizer> BodyRegexSanitizers => [
+        // Sanitizes all URLs to remove actual service names
+        new BodyRegexSanitizer(new BodyRegexSanitizerBody() {
+          Regex = "(?<=http://|https://)(?<host>[^/?\\.]+)",
+          GroupForReplace = "host",
+        })
+    ];
+
+    [Fact]
+    public async Task Should_get_storage_sync_service()
```

Comment:
Only 1 Live/Recorded Test is added. 
We need to add Recorded/Live test coverage for all 18 commands.
We're missing test coverage here.